### ISSUE TITLE
Helm chart broken since the introduction of watched_namespace.yaml in chart templates.

### ISSUE THREAD

**Issue by fredcoq** [druid-io/druid-operator#252]
I have a CD pipeline synchronizing with druid-operator repo.
Helm chart synchronization fails with the following error : 
```
executing "druid-operator/templates/watched_namespace.yaml" at <ne $relname.env.WATCH_NAMESPACE "default">: 
error calling ne: incompatible types for comparison
```
It can be reproduced by cloning the repo and performing  chart validation : 
Helm version 3.7.0
```
➜  druid-operator git:(master) helm lint ./chart/         
==> Linting ./chart/
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/: template: druid-operator/templates/watched_namespace.yaml:4:43: executing "druid-operator/templates/watched_namespace.yaml" at <ne $relname.env.WATCH_NAMESPACE "default">: error calling ne: incompatible types for comparison
```
Replacing $relname by .Value in watched_namespace.yaml restore the expected behavior when syncing with helm.

**Comment by achetronic** [comment]
> this should fix #253

Have you tested it? I did in that way pre-supposing that druid-operator would be a Helm dependency :)

**Comment by AdheipSingh** [comment]
yes i have tested it. added checks on CI too.

somehow index .Values.druidOperator values were nil were as we wanted string. so just kept is simple.

pls feel to comment/review if you feel anything is missing :) 

Thanks
